### PR TITLE
DBZ-842 Do not send TOASTed values from wal2json

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-842

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @jpechane. I've pushed one more commit to expand the test a bit (also covering NOT NULL column now).

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java [hunk 1]
```diff
@@ -254,7 +254,8 @@ public ChainedLogicalStreamBuilder optionsWithoutMetadata(ChainedLogicalStreamBu
             .withSlotOption("pretty-print", 1)
             .withSlotOption("write-in-chunks", 1)
             .withSlotOption("include-xids", 1)
-            .withSlotOption("include-timestamp", 1);
+            .withSlotOption("include-timestamp", 1)
+            .withSlotOption("include-unchanged-toast", 0);
```
Comment (murongruomeng) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: lastest wal2json does not support include-unchanged-toast parameter

**Comment by gunnarmorling** [comment]
Thanks for the heads-up! Will it fail when passing the option, or will it
be silently ignored?

**Comment by murongruomeng** [comment]
failed

**Comment by gunnarmorling** [comment]
Thanks; so for now the wal2json version supported by Debezium is 1.0, which doesn't expose this issue. That things continue to work when updating to latest commits from wal2json isn't something we can guarantee unfortunately. For this very case, I've filed https://issues.jboss.org/browse/DBZ-1053, which should be in 0.9.0.CR1 (early January).

**Comment by rpofuk** [comment]
Seems like wat2json does no support this options any more: 

https://github.com/eulerto/wal2json/commit/66d836d209f71f2337ea622ae63669d759e30926#diff-7cc6c09e7ed49666951d3bd508952c52

https://github.com/eulerto/wal2json/commit/2e6384cdfe9e97ece59596b2b46384ed6d248bba#diff-7cc6c09e7ed49666951d3bd508952c52

FROM Postgres 10.4 on AWS RDS I GET: 

org.apache.kafka.connect.errors.ConnectException: org.postgresql.util.PSQLException: ERROR: option \"include-unchanged-toast\" = \"0\" is unknown
  Where: slot \"log_collector_svc\", output plugin \"wal2json\", in the startup callback
	at io.debezium.connector.postgresql.RecordsStreamProducer.start(RecordsStreamProducer.java:119)
	at io.debezium.connector.postgresql.PostgresConnectorTask.start(PostgresConnectorTask.java:126)
	at io.debezium.connector.common.BaseSourceTask.start(BaseSourceTask.java:47)
	at org.apache.kafka.connect.runtime.WorkerSourceTask.execute(WorkerSourceTask.java:198)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:175)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:219)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.postgresql.util.PSQLException: ERROR: option \"include-unchanged-toast\" = \"0\" is unknown
  Where: slot \"log_collector_svc\", output plugin \"wal2json\", in the startup callback
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
	at org.postgresql.core.v3.QueryExecutorImpl.processCopyResults(QueryExecutorImpl.java:1116)
	at org.postgresql.core.v3.QueryExecutorImpl.writeToCopy(QueryExecutorImpl.java:998)
	at org.postgresql.core.v3.CopyDualImpl.writeToCopy(CopyDualImpl.java:19)
	at org.postgresql.core.v3.replication.V3PGReplicationStream.updateStatusInternal(V3PGReplicationStream.java:189)
	at org.postgresql.core.v3.replication.V3PGReplicationStream.forceUpdateStatus(V3PGReplicationStream.java:109)
	at io.debezium.connector.postgresql.connection.PostgresReplicationConnection.startPgReplicationStream(PostgresReplicationConnection.java:305)
	at io.debezium.connector.postgresql.connection.PostgresReplicationConnection.createReplicationStream(PostgresReplicationConnection.java:195)
	at io.debezium.connector.postgresql.connection.PostgresReplicationConnection.startStreaming(PostgresReplicationConnection.java:179)
	at io.debezium.connector.postgresql.RecordsStreamProducer.start(RecordsStreamProducer.java:105)
	... 10 more


Please revert!
